### ISSUE TITLE
Overview of reactions like in Facebook

### ISSUE THREAD

**Issue by conscho** [cornelltech/unicornucopia-issue-tracker#27]
I'm guessing that you already planned this: When hovering over the reactions that a post already received show a detailed list how many times which reaction has been given. The total number is not very descriptive, since some reactions are positive and some are negative.

**Comment by AVatch** [comment]
yup, you can click on the badges and actually see the breakdown, but thats to be done

**Comment by omris1** [comment]
+1
